mri/reconstructors/utils/extract_sensitivity_maps.py

```diff
@@ -49,7 +54,20 @@ def extract_k_space_center_and_locations(data_values, samples_locations,
             raise ValueError('target image cartesian image shape must be fill')
         raise NotImplementedError
     else:
-        data_thresholded = np.copy(data_values)
+        if data_values.ndim > 2:
+            warnings.warn('Data Values seems to have dimension '
+                          + str(data_values.ndim) +
+                          ' (>2). Using isFFT for now.')
+            isFFT = True
+        if isFFT:
+            img_shape = np.asarray(data_values[0].shape)
+            mask = convert_locations_to_mask(samples_locations, img_shape)
+            indices = np.where(np.reshape(mask, mask.size))[0]
+            data_thresholded = np.asarray([
```

Comment:
there is no thresholding applied yet right?

Reply:
Well I am just keeping common codes, this data_thresholded here just holds data in format similar to that from NFFT, so that thresholding can be applied. I can change names if needed only for this variable, but right now it is something that it will hold eventually

Reply:
yes but if it's not holding at this point in the code it's confusing.

Reply:
Changing

Reply:
Made it to data_ordered.